Week2/README.md

```diff
@@ -59,12 +59,6 @@ If you already have the table, and you just want to change a column to Primary K
 ALTER TABLE teachers ADD PRIMARY KEY (teacher_number);
 ```
 
-or you can define a primary key later:
-
-```sql
-ALTER TABLE teachers ADD CONSTRAINT PK_Person PRIMARY KEY (ID,LastName);
```

Comment:
this way of adding primary key is syntactically different that the previous one.  So I think it should be there for reference.

Reply:
You are correct that its (slightly) syntactically different but I propose to focus on concepts rather then different syntactical operations that do exactly the same thing.

Reply:
It is just one extra line which IMHO does not harm any conceptual explanation.

Reply:
It has been 15 years since I did any SQL like this so I may be wrong, but if it is only syntactically different then I would expect the two lines to at least make the same change to the database. Am I correct that these two lines, while doing a similar thing to the database in concept, do not make the exact same change to the database?

Reply:
The only difference is that the removed example adds a named constraint. Which is specifically useful in SQL Server, Oracle and MS Access. Because in those DB's you have to remove the PK by name. 

I think we should stick with teaching concepts rather than syntactical oddities

Reply:
I still disagree, but would like to move on. As long as students understand the concept, I am happy.
Let's remove it.